source/atlas-cli-changelog.txt

```diff
@@ -12,6 +12,50 @@
    :depth: 1
    :class: singlecol
 
+.. _atlas-cli-1.6.0:
+
+{+atlas-cli+} 1.6.0
+-------------------
+
+Released 30 March 2023
+
+- Adds the following new commands:
+
+  - :ref:`serverless-backups-snapshots-watch`
+  - :ref:`serverless-backups-restores-watch`
+  - :ref:`serverless-backups-restores-create`
+  - :ref:`serverless-backups-restores-describe`
+  - :ref:`atlas-serverless-backups-snapshots-list`
+  - :ref:`atlas-serverless-backups-restores-list`
+  - :ref:`atlas-serverless-update`
+  - :ref:`atlas-backups-exports-jobs-watch`
+
+- Adds the following to commands:
```

Comment:
It might make sense to list each of these individually (i.e. updates the <x> command to add ...). Think it might be easier for users to identify changes relevant to them